_posts/2023-02-06-ruby-girls-nepal-meetup-feb-2023.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Please change the file name as well to `_posts/2023-02-06-rails-girls-nepal-meetup-feb-2023.md`